### PR TITLE
feat: Add team_id tag to plugin-server sentry errors

### PR DESCRIPTION
**Pull request by tiina303**
## Problem

If we had team id info in sentry we could look this up for bug reports by tag

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

ran locally (https://posthog.sentry.io/settings/projects/karl-playground/keys/): 
```
SENTRY_DSN='<DNS-for-Karl-playground>' REGION='testing' DEBUG=1 ./bin/start
```

Example sentry error with team_id tag: https://posthog.sentry.io/issues/4166587036/?project=6605447&query=&referrer=issue-stream&statsPeriod=14d&stream_index=0

### CODE REVIEW

File: plugin-server/src/main/ingestion-queues/session-recording/blob-ingester/session-manager.ts [hunk 1]
```diff
@@ -83,6 +56,33 @@ export class SessionManager {
         // this.lastProcessedOffset = redis.get(`session-recording-last-offset-${this.sessionId}`) || 0
     }
 
+    private async deleteFile(file: string, context: string) {
```
Comment (hazzadous) [inline plugin-server/src/main/ingestion-queues/session-recording/blob-ingester/session-manager.ts:59]: Why the move!

File: plugin-server/src/worker/ingestion/event-pipeline/runner.ts [hunk 2]
```diff
@@ -52,7 +52,7 @@ export class EventPipelineRunner {
 
         try {
             let result: EventPipelineResult | null = null
-            const eventWithTeam = await this.runStep(populateTeamDataStep, [this, event])
+            const eventWithTeam = await this.runStep(populateTeamDataStep, [this, event], event.team_id || -1)
```
Comment (hazzadous) [inline plugin-server/src/worker/ingestion/event-pipeline/runner.ts:55]: team_id can be 0 right?

File: plugin-server/src/main/ingestion-queues/session-recording/blob-ingester/session-manager.ts [hunk 3]
```diff
@@ -83,6 +56,33 @@ export class SessionManager {
...
+                    }
+                )
+                return
+            }
+            status.error('🧨', `blob_ingester_session_manager failed deleting file ${context}path: ${file}`, {
+                err,
+                file,
+                context,
+            })
+            captureException(err)
```
Comment (tiina303) [inline plugin-server/src/main/ingestion-queues/session-recording/blob-ingester/session-manager.ts:81]: note: I'm missing the team_id tag here still  ... but I'll just put up a follow-up PR for that

**Review by hazzadous — APPROVED** [review]
Looks good just minor points. Hopefully the types are aligned with the actual data 🤞 

Maybe Ben or Paul could check the blob store?

**Pull request merged**